Training2/src/task1/src/Component/Modal/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react"
+import './Modal.css'
+
+export default function Modal(){
+    const [modal, setModal] = useState(false)
```

Comment:
the variable name can make misunderstanding, boolean variable should start with is, should...

isOpenModal, setIsOpenModal

Reply:
dạ vâng